ProteoformSuiteInternal/CytoscapeScript.cs

```diff
@@ -186,9 +188,11 @@ private static string get_script(int feature_count, bool quantitative, string ed
         {
             string tsv_header = "accession_1\t" + lysine_count_header + "\taccession_2\t" + delta_mass_header + "\t" + edge_ptm_header;
             string edge_rows = "";
-            foreach (ProteoformRelation r in families.SelectMany(f => f.relations))
+            foreach (ProteoformRelation r in families.SelectMany(f => f.relations).Distinct())
```

Comment:
I noticed because I was printing out lists of families to get #'s for a poster and there were repeats! I'm not sure if this completely solved this issue though, on my todo list to look into it again.